example-app/src/main/res/layout/activity_main.xml

```diff
@@ -7,34 +7,59 @@
     android:theme="@style/Theme.AppCompat"
     tools:context=".MainActivity">
 
-    <LinearLayout
-        android:orientation="horizontal"
+    <RadioGroup
+        android:id="@+id/sizeRadioGroup"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
-
-        <Button
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+        <RadioButton
+            android:id="@+id/size_invisible"
```

Comment:
I like the new configuration buttons. I do have this issue where the text is not visible.

Device: 
![image](https://user-images.githubusercontent.com/11096226/162219060-0517c151-b5ff-4562-ae91-c9bf4c3ab760.png)


![image](https://user-images.githubusercontent.com/11096226/162218953-0020f986-a974-4802-b7e4-8643debb3fc0.png)

Reply:
Fixed